### DISCUSSION TITLE
Running Apps in Production using Turborepo inside a single Docker Container

### DISCUSSION THREAD

**Discussion by subodhpareek18** [vercel/turborepo#775] [Help] [answered]
Hi, I have gone through the documentation, issues and the discussion threads here but wasn't able to find an answer.

I have a use case with 2 backend servers and a frontend webapp needing to run side by side. Running `turbo run dev` and `turbo run build` are clearly documented in all examples but if I wanted to run the built code in production by putting a `start` script in all individual packages and then put a `turbo run build && turbo run start` as a `serve` command on the top level package.json file and that run that inside a Docker container in production so all of the apps can be deployed and run together. This would ofcourse mean turbo is added a dependency and not a devDependency.

Is that a recommended way to go about it? Any inputs will be appreciated.

**Comment by gustavoOchoa** [comment]
i have the same problem.... i build my entire project put it on a docker but we cant find the "pnpm turbo run start" or whatever i need to run it in PM2.

**Comment by volnei** [comment]
Did you find a solution? 

Thanks

**Reply by gustavoOchoa** [reply, marked as answer]
yes just about 5 minutes ago:

to run a single app
```
turbo run start --filter=./apps/intranet
```
to run all aps:
```
turbo run start --filter=./apps/*
```
and in turbo.json u need to add the start script:
```
    "start": {
      "cache": true,
      "persistent": true
    }
```
this will run the script "start" in every app that u have in apps which had a start script like "next start".
Hope this helps anyone!
